jsk_fetch_robot/jsk_fetch_startup/euslisp/go-to-kitchen.l

```diff
@@ -29,7 +29,11 @@
                                      (success-go-to-kitchen . nil)
                                      (success-go-to-trashcan . nil)
                                      (success-auto-dock . nil))
-                              :hz 2.0)))
+                              :hz 2.0
+                              :before-hook-func
+                              '(lambda ()
+                                (set-alist 'description "" 'userdata)
+                                (set-alist 'image "" 'userdata)))))
```

Comment:
can you try like this with the latest `fetch15` branch.
I chnaged to pass userdata as argument.
[jsk-ros-pkg/jsk_roseus@`e5e739c` (#715)](https://github.com/jsk-ros-pkg/jsk_roseus/pull/715/commits/e5e739c9fccd45a6a3c047043ef21400f1884ae1)

```suggestion
                              '(lambda (userdata)
                                (set-alist 'description "" userdata)
                                (set-alist 'image "" userdata)))))
```